src/Tribe/Events/Attendees_List.php

```diff
@@ -153,7 +186,74 @@ public function maybe_update_attendee_list_hide_meta( $post ) {
 			$post
 		);
 
-		return update_post_meta( $post->ID, Tribe__Tickets_Plus__Attendees_List::HIDE_META_KEY, (int) $is_showing_attendee_list );
+		return update_post_meta( $post->ID, self::HIDE_META_KEY, (int) $is_showing_attendee_list );
+	}
+
+	/**
+	 * Get list of public attendees for display.
+	 *
+	 * @since TBD
+	 *
+	 * @param WP_Post|int $post_id Post object or ID.
+	 * @param  int        $limit   Limit of attendees to be retrieved from database.
+	 *
+	 * @return array List of public attendees for display.
+	 */
+	public function get_attendees_for_post( $post_id, $limit = - 1 ) {
+		$post = get_post( $post_id );
+
+		if ( ! $post instanceof WP_Post ) {
+			return [];
+		}
+
+		$args = [
+			'by' => [
+				// Exclude people who have opted out or not specified optout.
+				'optout' => 'no_or_none',
+				// Only include public attendees.
+				'post_status' => 'publish',
```

Comment:
:warning: Array double arrow not aligned correctly; expected 10 space(s) between "'post_status'" and double arrow, but found 1.

<sub>hash: 59020115341227b5a765086b50904fe4</sub>